frontend/src/pages/AccountSend.vue

```diff
@@ -393,6 +716,8 @@ function useSendForm() {
       } else if (nativeTokenValue.chainId !== prevNativeTokenValue.chainId || isLoadingValue !== prevIsLoadingValue) {
         // When network finally connects after page load, we need to re-parse the payment link data.
         await setPaymentLinkData(); // Handles validations.
+        batchSendIsSupported.value = batchSendSupportedChains.includes(Number(currentChain.value?.chainId));
```

Comment:
chainId is a `uint256` so let's treat it as such and avoid using `Number()` to cast. In practice this won't matter for us because (1) we only support a few chains and all have chain IDs smaller than `Number.MAX_SAFE_INTEGER` and (2) AFAIK no chains currently have a chain ID that big anyway, but better to be cautious
```suggestion
        const chainId = BigNumber.from(currentChain.value?.chainId).toNumber()
        batchSendIsSupported.value = batchSendSupportedChains.includes(chainId);
```

Reply:
Good to know, fixed it here https://github.com/ScopeLift/umbra-protocol/pull/491/commits/a3c2970279e53167566ddb40bc6d834ea7eefbbe